mono/metadata/mono-perfcounters.c

```diff
@@ -1361,15 +1364,17 @@ mono_perfcounter_get_impl (MonoString* category, MonoString* counter, MonoString
 	case CATEGORY_SECURITY:
 	case CATEGORY_ASPNET:
 	case CATEGORY_THREADPOOL:
-		result = predef_writable_get_impl (cdesc->id, counter, c_instance, type, custom);
+		result = predef_writable_get_impl (cdesc->id, counter, counter_length, c_instance, type, custom);
 		break;
 	}
 	g_free (c_instance);
 	return result;
 }
 
 MonoBoolean
-mono_perfcounter_get_sample (void *impl, MonoBoolean only_value, MonoCounterSample *sample)
+ves_icall_System_Diagnostics_PerformanceCounter_GetSample (
+	void *impl, MonoBoolean only_value, MonoCounterSample *sample, MonoError *error)
+// previously mono_perfcounter_get_sample
```

Comment:
Why keeping that?

Reply:
It is much easier to sere here than in history and it has been useful looking at other branches at the same time.